modules/jwplayerBidAdapter.js

```diff
@@ -50,4 +87,129 @@ export const spec = {
+    deepSetValue(openrtbRequest, 'regs.ext.us_privacy', bidderRequest.uspConsent);
+  }
+
+  return JSON.stringify(openrtbRequest);;
+}
+
+function buildRequestImpression(bidRequest) {
+  const impressions = [];
+
+  const impressionObject = {
+    id: bidRequest.adUnitCode,
+    secure: isSecure() ? 1 : 0
+  };
+
+  impressionObject.video = buildImpressionVideo(bidRequest);
+
+  const bidFloorData = buildBidFloorData(bidRequest);
+  impressionObject.bidfloor = bidFloorData.floor;
+  impressionObject.bidfloorcur = bidFloorData.currency;
+  
+  impressionObject.ext = buildImpressionExtension(bidRequest);
+
+  impressions.push(impressionObject);
+
+  return impressions;
+}
+
+function buildImpressionVideo(bidRequest) {
+  const videoParams = deepAccess(bidRequest, 'mediaTypes.video', {});
+
+  const video = {};
+
+  // Obtain all ORTB params related video from Ad Unit
+  VIDEO_ORTB_PARAMS.forEach((param) => {
+    if (videoParams.hasOwnProperty(param)) {
+      video[param] = videoParams[param];
+    }
+  });
+
+  return video;
+}
+
+function buildImpressionExtension(bidRequest) {
+  return {
+    appnexus: {
+      placement_id: bidRequest.params.placementId
+    }
+  };
+}
+
+function buildBidFloorData(bidRequest) {
+  const {params} = bidRequest;
+  // Bid Floor
+  const bidFloorRequest = {
+    currency: params.currency || 'USD',
+    mediaType: 'video',
+    size: '*'
+  };
+
+  let floorData;
+  if (isFn(bidRequest.getFloor)) {
+    floorData = bidRequest.getFloor(bidFloorRequest);
+  } else if (params.bidfloor) {
+    floorData = {floor: params.bidfloor, currency: params.currency || 'USD'};
+  }
+
+  return floorData;
+}
+
+function buildRequestSite(bidRequest) {
+  const site = {
+    domain: window.location.hostname,
+    page: window.location.href,
+    ref: bidRequest.refererInfo ? bidRequest.refererInfo.referer || null : null
+  };
+
+  const videoParams = deepAccess(bidRequest, 'mediaTypes.video', {});
+
+  // Site Content
+  if (videoParams.content && isPlainObject(videoParams.content)) {
+    openrtbRequest.site.content = {};
+    const contentStringKeys = ['id', 'title', 'series', 'season', 'genre', 'contentrating', 'language', 'url'];
+    const contentNumberkeys = ['episode', 'prodq', 'context', 'livestream', 'len'];
+    const contentArrayKeys = ['cat'];
+    const contentObjectKeys = ['ext'];
+    for (const contentKey in videoBidderParams.content) {
+      if (
+        (contentStringKeys.indexOf(contentKey) > -1 && isStr(videoParams.content[contentKey])) ||
+        (contentNumberkeys.indexOf(contentKey) > -1 && isNumber(videoParams.content[contentKey])) ||
+        (contentObjectKeys.indexOf(contentKey) > -1 && isPlainObject(videoParams.content[contentKey])) ||
+        (contentArrayKeys.indexOf(contentKey) > -1 && isArray(videoParams.content[contentKey]) &&
+        videoParams.content[contentKey].every(catStr => isStr(catStr)))) {
+        site.content[contentKey] = videoParams.content[contentKey];
```

Comment:
`videoPrams.content[contentKey]` is terrible for minification and it's repeated 6 times, which means you're adding a lot of bloat.
`videoPrams.content[contentKey]` minifies to `a.content[contentKey]` which is very bad. 

Instead, if you use a var, the entire var gets minified to a single letter.
`const contentValue = videoPrams.content[contentKey]` minifies to `var a = videoPrams.content[contentKey]`
and then `a` is repeated 6 times, saving you a ton of chars.

Reply:
removed